salt/states/zpool.py

```diff
@@ -30,6 +30,23 @@
               /dev/disk2
               /dev/disk3
 
+    partitionpool:
+      zpool.present:
+        - config:
+            import: false
+            force: true
+        - properties:
+            comment: disk partition salty storage pool
+            ashift: '12'
+            feature@lz4_compress: enabled
+        - filesystem_properties:
+            compression: lz4
+            atime: on
+            relatime: on
+        - layout:
+            disk-0:
```

Comment:
Since the example I've put in the commit is for a _single partition pool_, what is the correct definition?  In my last message, I based the first attempt on the `simplepool` example and ended up with errors (as shown in that message).